etc/ubsansup.txt

```diff
@@ -1,6 +1,2 @@
-# bmqp::Crc32c carries out misaligned access of Int64 in one of the internal
-# routines, but only on x86/64.  Misaligned accesses are handled gracefully by
-# this hardware, unlike SPARC.  So we simply suppress this warning.  Other
-# option is to update the code, which can be done by someone feeling
-# adventurous.
-alignment:crc32c1024SseInt
```

Comment:
According to comment the problem appears on SPARC. Since sanitizers are run on x86/64, removed it